### PR TITLE
updated doc

### PR DESCRIPTION
**Pull request by punit-squadcast**

### CODE REVIEW

File: docs/resources/deduplication_rules.md [hunk 1]
```diff
@@ -13,17 +13,17 @@ description: |-
...
-  name = "test"
+data "squadcast_service" "example_resource_name" {
+  name = "example service name"
 }
 
-resource "squadcast_deduplication_rules" "test" {
-  team_id    = data.squadcast_team.example.id
-  service_id = data.squadcast_service.example.id
+resource "squadcast_deduplication_rules" "example_resource_name" {
+  team_id    = data.squadcast_team.exaexample_resource_namemple.id
```
Comment (raghuchinnannan) [inline docs/resources/deduplication_rules.md]: > exaexample_resource_namemple.id

Can you fix this @punit-squadcast ?

Comment (punit-squadcast) [reply docs/resources/deduplication_rules.md]: sure

Comment (punit-squadcast) [reply docs/resources/deduplication_rules.md]: Done @raghuchinnannan

File: docs/resources/team_role.md [hunk 2]
```diff
@@ -43,6 +45,6 @@ resource "squadcast_team_role" "test" {
 Import is supported using the following syntax:
 
 ```shell
-# teamID:roleID
-terraform import squadcast_team_role.test 62d2fe23a57381088224d726:62da76c088f407f9ca756ca5
+# teamID:teamRole(ed: Admin, User, Observer)
```
Comment (raghuchinnannan) [inline docs/resources/team_role.md]: > teamRole(ed: Admin, User, Observer)

Should be

> teamRole (ex: Admin, User, Observer)

File: examples/resources/squadcast_team_role/import.sh [hunk 3]
```diff
@@ -1,2 +1,2 @@
-# teamID:roleID
-terraform import squadcast_team_role.test 62d2fe23a57381088224d726:62da76c088f407f9ca756ca5
\ No newline at end of file
+# teamID:teamRole(ed: Admin, User, Observer)
```
Comment (raghuchinnannan) [inline examples/resources/squadcast_team_role/import.sh]: > teamID:teamRole(ed: Admin, User, Observer)

Should  be

> teamID:teamRole (ex: Admin, User, Observer)

**Review by raghuchinnannan — COMMENTED** [review]
@punit-squadcast Please make the commented changes.

File: examples/resources/squadcast_team_role/import.sh (hunk 3 above)
Comment (punit-squadcast) [reply examples/resources/squadcast_team_role/import.sh]: Done @raghuchinnannan

File: examples/resources/squadcast_team_role/import.sh [hunk 4]
```diff
@@ -1,2 +1,2 @@
-# teamID:roleID
-terraform import squadcast_team_role.test 62d2fe23a57381088224d726:62da76c088f407f9ca756ca5
\ No newline at end of file
+# teamID:teamRole(ex: Admin, User, Observer)
+terraform import squadcast_team_role.example_resource_name 62d2fe23a57381088224d726:Admin
```
Comment (raghuchinnannan) [inline examples/resources/squadcast_team_role/import.sh:2]: @punit-squadcast It should be an ID right? Not Role name?

Comment (punit-squadcast) [reply examples/resources/squadcast_team_role/import.sh:2]: no for role import we need to give name, I tried with Id but it was throwing error, it was working fine with name.

**Review by raghuchinnannan — APPROVED** [review]

**Review by raghuchinnannan — APPROVED** [review]

**Pull request merged**